docs/components.md

```diff
@@ -173,7 +173,7 @@ functionality is not provided by kubernetes itself.
 We will try to leverage as much of Kubernetes regarding to mounting and preparing images for VMI.
 However, `virt-handler` may provide a plugin mechanism to allow storage mounting and setup from the host, if the KubeVirt requirements do not fit into the Kubernetes storage scenarios.
 
-Since host side preparation of storage may not be enough, a cluster-wide [Storage Controller](###storage-controller) can be used to prepare storage.
```

Comment:
The Storage Controller link has expired, so I changed it to an absolute link. the Network Controller also fails.

Reply:
according to [1] a single hash should work for relative links, did you try that?

[1] https://gist.github.com/asabaylus/3071099

Reply:
Sorry to keep you waiting, I have updated it.